jwst/refpix/refpix_step.py

```diff
@@ -17,12 +17,12 @@ class RefPixStep(Step):
     class_alias = "refpix"
 
     spec = """
-        odd_even_columns = boolean(default=True)
-        use_side_ref_pixels = boolean(default=True)
-        side_smoothing_length = integer(default=11)
-        side_gain = float(default=1.0)
-        odd_even_rows = boolean(default=True)
-        ovr_corr_mitigation_ftr = float(default=1.8)
+        odd_even_columns = boolean(default=True) # Compute reference signal separately for even/odd columns (NIR only)
```

Comment:
The config parser doesn't like having ()'s in the comment (it thinks it's another arg definition). Look at all the CI failures.